src/DevBetterWeb.Web/Pages/Admin/Invitations/Index.cshtml.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoMapper;
+using DevBetterWeb.Core;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Web.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace DevBetterWeb.Web.Pages.Admin.Invitations;
+
+[Authorize(Roles = AuthConstants.Roles.ADMINISTRATORS)]
+public class IndexModel : PageModel
+{
+	public IList<InvitationDto> Invitations { get; set; } = new List<InvitationDto>();
+	
+  private readonly IRepository<Invitation> _invitationRepository;
+  private readonly IMapper _mapper;
+
+  public IndexModel(IRepository<Invitation> invitationRepository, IMapper mapper)
+  {
+    _invitationRepository = invitationRepository;
+    _mapper = mapper;
+  }
+
+  public async Task<IActionResult> OnGetAsync()
+  {
+	  var invitations = await _invitationRepository.ListAsync();
```

Comment:
I might want a way to filter to just active invites but we'll stick with this for now while I see how it works.

Reply:
yes, it has a filter same as I did on members.
![image](https://user-images.githubusercontent.com/6225593/178764496-6dee0397-6468-40bb-90e7-d690065e26fb.png)